wsimod/nodes/nodes.py

```diff
@@ -1142,6 +1173,18 @@ def __init__(self, decays={}, parent=None, **kwargs):
         self.end_timestep = self.end_timestep_decay
         self.ds = self.decay_ds
 
+    def apply_overrides(self, overrides: Dict[str, Any] = {}):
+        """Apply overrides to the decaytank.
+    
+        Enables a user to override any of the following parameters: 
+        decays.
+        
+        Args:
+            overrides (dict, optional): Dictionary of overrides. Defaults to {}.
+        """
+        self.decays.update(overrides.pop("decays", {}))
```

Comment:
```suggestion
        self.decays.update(overrides.get("decays", {}))
```

Reply:
Again, if `decays` is not provided in `overrides`, then this will replace existing decays:
```suggestion
        self.decays.update(overrides.pop("decays", self.decays))
```